lib/protocol/src/openctx-protocol.schema.ts

```diff
@@ -62,12 +76,38 @@ export interface Selector {
      */
     contentContains?: string
 }
-export interface ItemsParams {
+/**
+ * TODO: move this to openctx.schema.json and figure out why it is not getting imported in the protocol-schema.ts
+ */
+export interface Mention {
+    /**
+     * A descriptive title.
+     */
+    title: string
+    /**
+     * A URI for the mention item.
+     */
+    uri?: string
```

Comment:
in the slack thread I mentioned making this a requirement. but I guess given you added data it doesn't have to be required.

Reply:
Updated, in the json schema, I made a typo, due to which it was left optional. Lets keep it required only.